tests/framework/http.py

```diff
@@ -22,7 +22,27 @@ def is_good_response(response: int):
             """Return `True` for all HTTP 2xx response codes."""
             return 200 <= response < 300
 
+        def is_status_ok(response: int):
```

Comment:
nit: you have a typo in this commit message as well. Sorry, I didn't notice (/s/actutual/actual).

Reply:
Oh, sorry. Thanks!